.travis.yml

```diff
@@ -1,13 +1,45 @@
 sudo: false
 language: node_js
-script:
-  - npm run build
-  - npm run test:ci
 node_js:
+  - 6
   - 8
   - 10
   - 12
 
+before_install:
+  - npm config set depth 0
+before_script: >
+  node_version=$(node -v);
+  if [ ${node_version:3:1} = "." ]; then
+    echo "Node 10+"
+    npm install --no-save "eslint@6"
+    npm run build
+    npm run test:ci
+    else
+      if [ ${node_version:1:1} -ge 8 ]; then
+        echo "Node 8+"
+        npm install --no-save "eslint@5"
+        npm run build
+        npm run test:ci
+      else
+        if [ ${node_version:1:1} -ge 6 ]; then
+          echo "Node 6+"
+          npm install --no-save "eslint@5" mocha@6.2.2 nyc@14.1.1
+          npm run test:ci
+        else
+          if [ ${node_version:1:1} -ge 4 ]; then
```

Comment:
This looks like dead code since we only list node 6, 8, 10, 12 in the config above.

Reply:
Gotcha, fixed (I can't confirm Travis builds are ok now as the site is not loading for me here).